CONTRIBUTING.md

```diff
@@ -13,14 +13,98 @@ Check out the [Projects](https://github.com/jMonkeyEngine/jmonkeyengine/projects
 
 When you're ready to submit your code, just make a [pull request](https://help.github.com/articles/using-pull-requests).
 
-- Do not commit your code until you have received proper feedback.
 - In your commit log message, please refer back to the originating forum thread (example) for a ‘full circle’ reference. Also please [reference related issues](https://help.github.com/articles/closing-issues-via-commit-messages) by typing the issue hashtag.
 - When committing, always be sure to run an update before you commit. If there is a conflict between the latest revision and your patch after the update, then it is your responsibility to track down the update that caused the conflict and determine the issue (and fix it). In the case where the breaking commit has no thread linked (and one cannot be found in the forum), then the contributor should contact an administrator and wait for feedback before committing.
 - If your code is committed and it introduces new functionality, please edit the wiki accordingly. We can easily roll back to previous revisions, so just do your best; point us to it and we’ll see if it sticks!
```

Comment:
Also a misuse of 'commit'. This should be 'merged'.

Reply:
Maybe. But I merge my own PRs on a regular basis, usually without "proper feedback". Perhaps the whole notion is outdated.

Reply:
I was not mentioning the line you removed. (Though it is rather prominent in the *wide* sample that GH so *helpfuly* provides.) I was talking about the sentence that starts "If your code is committed and it introduces new functionality,"

Reply:
You're right, but again, I think that should be a separate PR.